CONTRIBUTING.md

```diff
@@ -89,6 +89,33 @@ TURBO_BINARY_PATH=~/repos/vercel/turbo/cli/turbo.exe npm link turbo
 
 If you're using a different package manager replace npm accordingly.
 
+## Manually testing `turbo`
+
+Before releasing, it's recommended to test the `turbo` binary manually.
+Here's a checklist of testing strategies to cover:
+
+- Test `login`, `logout`, `login --sso-team`, `link`, `unlink`
+- Test every command with `--skip-infer`.
```

Comment:
Is it necessary to test this codepath separate from the global only installation scenario?
(Calling this out as it brings down the test matrix this section describes from 60 tests to 30)

Reply:
Yeah probably not